child_compassion/model/child_compassion.py

```diff
@@ -328,6 +338,20 @@ def _get_last_pictures(self, cr, uid, child_id, context=None):
 
         return pic_id
 
+    def _create_empty_case_study(self, cr, uid, child, context=None):
+        child_prop_obj = self.pool.get('compassion.child.property')
+
+        if not (child_prop_obj.search(
+                cr, uid,
+                [('child_id', '=', child.id)],
+                context=context)):
+            vals = {
+                'child_id': child.id,
+                'info_date': date.today()
+            }
+            child_prop_obj.create(cr, uid, vals, context)

```

Comment:
You could return this line instead of always returning True (and who knows, id of created object may be useful outside of the function).